include/nsysnet/_socket.h

```diff
@@ -0,0 +1,159 @@
+#pragma once
+#include <wut.h>
+#include <stdint.h>
+#include <sys/time.h>
+#include <sys/types.h>
+#include <sys/socket.h>
+#include <netinet/in.h>
+#include <arpa/inet.h>
+
+/**
+ * \defgroup nsysnet_socket Socket
+ * \ingroup nsysnet
+ * @{
+ */
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+#define NSYSNET_C(func) __rplwrap_##func
```

Comment:
I believe there's already an `RPLWRAP` macro that does this: https://github.com/devkitPro/wut/blob/master/include/wut_rplwrap.h

Reply:
Hadn't noticed, thanks!